gpytorch/means/fkl_means.py

```diff
@@ -0,0 +1,48 @@
+import torch
+import gpytorch
+from torch.nn.functional import softplus
+from gpytorch.priors import NormalPrior
+
+class LogRBFMean(gpytorch.means.Mean):
```

Comment:
You define two classes called `LogRBFMean`, here and in `means/quadratic_mean.py`

Reply:
Thanks! This one was supposed to be removed.